client/src/AppBundle/Service/Client/Sirius/SiriusApiGatewayClient.php

```diff
@@ -91,49 +79,37 @@ public function sendReportPdfDocument(SiriusDocumentUpload $upload, string $cont
      * @return mixed|\Psr\Http\Message\ResponseInterface
      * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    public function sendSupportingDocument(SiriusDocumentUpload $upload, string $content, string $submissionUuid, string $caseRef)
+    public function sendSupportingDocument(SiriusDocumentUpload $upload, string $submissionUuid, string $caseRef)
     {
-        $reportJson = $this->serializer->serialize(['data' => $upload], 'json');
-
-        $multipart = new MultipartStream([
-            [
-                'name' => 'supporting_document',
-                'contents' => $reportJson
-            ],
-            [
-                'name' => 'supporting_document_file',
-                'contents' => base64_encode($content)
-            ],
-        ]);
+        $reportJson = $this->serializer->serialize(['supporting_document' => ['data' => $upload]], 'json');
 
         $signedRequest = $this->buildSignedRequest(
             sprintf(self::SIRIUS_SUPPORTING_DOCUMENTS_ENDPOINT, $caseRef, $submissionUuid),
             'POST',
-            $multipart,
-            'application/vnd.opg-data.v1+json',
-            'multipart/form-data'
+            $reportJson,
+            'application/vnd.opg-data.v1+json'
         );
 
-        return $this->httpClient->send($signedRequest);
+        return $this->httpClient->send($signedRequest, ['debug' => true]);
```

Comment:
Probably don't still want debug mode enabled?
```suggestion
        return $this->httpClient->send($signedRequest);
```